### PR TITLE
Use CLI installer for OpenCart install

### PR DESCRIPTION
**Pull request by TomAbel**
Replace data prefilling with sed in favor of the CLI installer.
Still needs work on 1.5.6.4

### CODE REVIEW

File: opencart 1.5.6.4/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        --username admin \
+        --password smailydev1 \
+        --email testing@smaily.sb \
+        --agree_tnc yes \
+        --http_server http://127.0.0.1:8080/
+    rm -rd ../install/
+fi
+echo 'OpenCart installed!'
+
+if [ ! -d $MAGE_ROOT_DIR/.modman ] ; then
```
Comment (kaittodesk) [inline opencart 1.5.6.4/entrypoint.sh]: What is `$MAGE_ROOT_DIR`? Where does it come from?

Comment (kaittodesk) [reply opencart 1.5.6.4/entrypoint.sh]: OK. It comes from `docker-compose.yml`.

Can it be renamed to `$OC_ROOT_DIR` or is `$MAGE_ROOT_DIR` required by modman?

Comment (TomAbel) [reply opencart 1.5.6.4/entrypoint.sh]: it's /var/www/html
specified in docker-compose for app

Comment (TomAbel) [reply opencart 1.5.6.4/entrypoint.sh]: No not required, I think it's just a weirdly named variable.

**Pull request merged**